commons/c2cgeoportal_commons/__init__.py

```diff
@@ -1,6 +1,12 @@
 """c2cgeoportal_commons package."""
 
-from c2cgeoportal_commons.models import get_session_factory, get_engine, get_tm_session, generate_mappers, init_dbsessions
+from c2cgeoportal_commons.models import (  # noqa
```

Comment:
Why noqa?

Reply:
Why do you add this here ? it is unused, so it do not pass checks with noqa

Reply:
I've supposed you add this imports here for use in c2cgeoportal, as I do not want to break c2cgeoportal, I've prefered to put noqa than to break something in c2cgeoportal.

Reply:
No, they should be removed :-)